lib/parser/lexer.rl

```diff
@@ -621,11 +621,13 @@ class Parser::Lexer
     ''   % { @num_xfrm = lambda { |chars| emit(:tINTEGER,   chars) } }
   | 'r'  % { @num_xfrm = lambda { |chars| emit(:tRATIONAL,  Rational(chars)) } }
```

Comment:
Do we not have the same issue with `1rescue`?

Reply:
Thanks, fixed.